model/relation/repository/rdf/RdfMediaRelationRepository.php

```diff
@@ -227,6 +229,25 @@ private function findAllMediaByTarget(string $targetId, string $type): MediaRela
         return $this->mapSourceRelations($type, (array)$result, $targetId);
     }
 
+    public function getItemAssetUris(string $itemUri): array
+    {
+        /** @var PersistenceManager $persistenceManager */
+        $persistenceManager = $this->getServiceManager()->get(PersistenceManager::SERVICE_ID);
+
+        /** @var common_persistence_SqlPersistence $persistence */
+        $persistence = $persistenceManager->getPersistenceById('default');
```

Comment:
As a suggestion, we can create a private method just to factory the connection: 

```
private function getPersistence():
{
    /** @var PersistenceManager $persistenceManager */
    $persistenceManager = $this->getServiceManager()->get(PersistenceManager::SERVICE_ID);

    /** @var common_persistence_SqlPersistence $persistence */
    return $persistenceManager->getPersistenceById('default');
}
```

```suggestion
        $persistence = $this->getPersistence();
```

Reply:
Already done.